### PR TITLE
[FSDE-12154] Migrate Pagination component examples to new documentation

### PR DESCRIPTION
**Pull request by josecarlosgonzalezv**

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-274/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://307121a770f283f539effedfd461e075e91f3ed7-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-274-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-274/2/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://bbd88bc6b0ba09c7e24f5686bb62e0d3f2372d56-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-274-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-274/5/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://80aa223d263fa426610d71d846934e05e22e509c-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-274-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-274/6/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://f80eff8be7baa294f3109247a7a450027a960bcf-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-274-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-274/7/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-274/8/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://d9d6e34c33260c2385086c1eff28430f6985ecbf-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-274-ux-chi.kubeodc-test.corp.intranet/ (internal)

File: src/documentation/pages/components/pagination/examples/_additional-sizes.vue [hunk 1]
```diff
@@ -0,0 +1,245 @@
+<template lang="pug">
+  <ComponentExample title="Additional Sizes" id="additional_sizes" :tabs="exampleTabs" padding="-p--0">
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/_additional-sizes.vue:2]: i'd recommend using dashes additional-sizes as per html id standards

File: src/documentation/pages/components/pagination/examples/_additional-sizes.vue [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+    p.-text(slot="example-description")
+      | Pagination supports the following sizes: <code>xs</code>, <code>sm</code>, <code>md</code> (default), <code>lg</code>, and <code>xl</code>.
+    <Wrapper slot="example">
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/_additional-sizes.vue:5]: no need for wrapper here, slot="example" could go in the below div as it's 1 child

File: src/documentation/pages/components/pagination/examples/_additional-sizes.vue [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+<ChiPagination :pages="5" :currentPage="3" size="sm" :inverse="true" />
+<ChiPagination :pages="5" :currentPage="3" size="md" :inverse="true" />
+<ChiPagination :pages="5" :currentPage="3" size="lg" :inverse="true" />
+<ChiPagination :pages="5" :currentPage="3" size="xl" :inverse="true" />`
+      }
+    };
+  }
+})
+export default class AdditionalSizes extends Vue {}
+</script>
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/_additional-sizes.vue:245]: looks like this file needs a new line at the end

File: src/documentation/pages/components/pagination/examples/_base.vue [hunk 4]
```diff
@@ -0,0 +1,118 @@
+<template lang="pug">
+  <ComponentExample title="Base" id="base" :tabs="exampleTabs" padding="-p--0">
+    <Wrapper slot="example">
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/_base.vue:3]: same here and the rest of cases like this

File: src/documentation/pages/components/pagination/examples/_base.vue [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+<ChiPagination :pages="5" :currentPage="3" />
+
+<!-- For dark backgrounds -->
+<ChiPagination :pages="5" :currentPage="3" :inverse="true" />`
+      }
+    };
+  }
+})
+export default class Base extends Vue {}
+</script>
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/_base.vue:118]: new line

File: src/documentation/pages/components/pagination/examples/_double-truncation.vue [hunk 6]
```diff
@@ -0,0 +1,123 @@
+<template lang="pug">
+  <ComponentExample title="Double Truncation" id="double_truncation" :tabs="exampleTabs" padding="-p--0">
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/_double-truncation.vue:2]: same here

File: src/documentation/pages/components/pagination/examples/_double-truncation.vue [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+<ChiPagination :pages="12" :currentPage="6" />
+
+<!-- For dark backgrounds -->
+<ChiPagination :pages="12" :currentPage="6" :inverse="true" />`
+      }
+    };
+  }
+})
+export default class DoubleTruncation extends Vue {}
+</script>
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/_double-truncation.vue:123]: new line

File: src/documentation/pages/components/pagination/examples/_results-label.vue [hunk 8]
```diff
@@ -0,0 +1,127 @@
+<template lang="pug">
+  <ComponentExample title="Results Label" id="results_label" :tabs="exampleTabs" padding="-p--0">
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/_results-label.vue:2]: same

File: src/documentation/pages/components/pagination/examples/_results-label.vue [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+<ChiPagination :pages="12" :currentPage="3" :results="240" />
+
+<!-- For light backgrounds -->
+<ChiPagination :pages="12" :currentPage="3" :results="240" :inverse="true" />`
+      }
+    };
+  }
+})
+export default class ResultsLabel extends Vue {}
+</script>
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/_results-label.vue:127]: same

File: src/documentation/pages/components/pagination/examples/portal/_base.vue [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+    </div>
+    <div class="chi-pagination__end"></div>
+  </div>
+</nav>`,
+        vue: `<ChiPagination :pages="3" :currentPage="2" :compact="true" :firstLast="true" size="xs" />`
+      }
+    };
+  }
+})
+export default class Base extends Vue {}
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/portal/_base.vue:80]: should be named BasePortal

File: src/documentation/pages/components/pagination/examples/index.vue [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+      <Truncation />
+      <DoubleTruncation />
+      <ResultsLabel />
+      <PageSize />
+      <PageJumper />
+      <Compact />
+      <AdditionalSizes />
+      
+    div(v-if="$store.state.themes.theme === 'portal'")
+      <Portal />
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/index.vue:16]: should be imported as BasePortal

File: src/documentation/pages/components/pagination/examples/portal/_results-label.vue [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+      </div>
+    </div>
+    <div class="chi-pagination__end"></div>
+  </div>
+</nav>`,
+        vue: `<ChiPagination :pages="3" :results="240" :currentPage="2" :compact="true" :firstLast="true" size="xs" />`
+      }
+    };
+  }
+})
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/portal/_results-label.vue:81]: should be named ResultsLabelPortal

File: src/documentation/pages/components/pagination/index.vue [hunk 13]
```diff
@@ -0,0 +1,36 @@
+<template lang="pug">
+  <div>
```
Comment (evansito) [inline src/documentation/pages/components/pagination/index.vue:2]: could be written as pug's div (without <>)

**Review by evansito — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-274/9/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://3573bd2a71b119795d99aca9435589550386f961-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-274-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-274/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://5f5cd73c181e2d9d57f8e4250fc00acab7292252-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-274-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-274/4/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://6f278a73991e03eff992f045730f9dcfcfb80866-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-274-ux-chi.kubeodc-test.corp.intranet/ (internal)

File: src/documentation/pages/components/pagination/examples/_additional-sizes_lumen_centurylink.vue [hunk 14]
```diff
@@ -0,0 +1,243 @@
+<template lang="pug">
+  <ComponentExample title="Additional Sizes" id="additional_sizes_lumen_centurylink" :tabs="exampleTabs" padding="-p--0">
```
Comment (soujanreddy) [inline src/documentation/pages/components/pagination/examples/_additional-sizes_lumen_centurylink.vue:2]: To maintain consistency, you can update to kebab-case for id's. I see we are using kebab-case through out the code.

File: src/documentation/pages/components/pagination/examples/_base_lumen_centurylink.vue [hunk 15]
```diff
@@ -0,0 +1,116 @@
+<template lang="pug">
+  <ComponentExample title="Base" id="base_lumen_centurylink" :tabs="exampleTabs" padding="-p--0">
```
Comment (soujanreddy) [inline src/documentation/pages/components/pagination/examples/_base_lumen_centurylink.vue:2]: Same here

File: src/documentation/pages/components/pagination/examples/_compact/_base_lumen_centurylink.vue [hunk 16]
```diff
@@ -0,0 +1,123 @@
+<template lang="pug">
+  <ComponentExample title="Base" id="base_compact_lumen_centurylink" :tabs="exampleTabs" padding="-p--0" titleSize="h4">
```
Comment (soujanreddy) [inline src/documentation/pages/components/pagination/examples/_compact/_base_lumen_centurylink.vue:2]: You can update for all id's.

**Review by soujanreddy — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-274/5/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://4187abeb04f83da7072947f2f97e5347fe688742-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-274-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-274/7/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-274/8/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-274/9/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://75de984c78a498728cbaa3b9059289f25b8bd417-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-274-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-274/10/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://bd3a34de434ff8f635114d1ffa29b9d778436ea1-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-274-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-274/11/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://b115ff39266bb2290025109d0ff894945a3d3668-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-274-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-274/12/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://1ff52f95556a9be3f03a50162bb86310da3f9790-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-274-ux-chi.kubeodc-test.corp.intranet/ (internal)

File: src/documentation/pages/components/pagination/examples/_compact/_page-jumper_lumen_centurylink.vue [hunk 17]
```diff
@@ -0,0 +1,89 @@
...
+          label: 'HTML blueprint'
+        }
+      ],
+      codeSnippets: {
+        webcomponent: `<!-- For light backgrounds -->
+<chi-pagination pages="3" current-page="2" compact page-jumper></chi-pagination>
+
+<!-- For dark backgrounds -->
+<chi-pagination pages="3" current-page="2" compact page-jumper inverse></chi-pagination>`,
+        htmlblueprint: `<!-- For light backgrounds -->
```
Comment (jllr) [inline src/documentation/pages/components/pagination/examples/_compact/_page-jumper_lumen_centurylink.vue:49]: Missing dark backgrounds example

File: src/documentation/pages/components/pagination/examples/_page-size_lumen_centurylink.vue [hunk 18]
```diff
@@ -0,0 +1,145 @@
+<template lang="pug">
+  <ComponentExample title="Page Size" id="page-size-lumen-centurylink" :tabs="exampleTabs" padding="-p--0">
+    p.-text(slot="example-description")
+      | Allow users to configure the number of result items per page.
+    .chi-grid.-no-gutter(slot="example")
+      .chi-col.-w--12
+        .-p--3
+          chi-pagination(pages="12" current-page="3" page-size)
```
Comment (jllr) [inline src/documentation/pages/components/pagination/examples/_page-size_lumen_centurylink.vue:8]: Missing results in page size

File: src/documentation/pages/components/pagination/examples/_portal/_results-label.vue [hunk 19]
```diff
@@ -0,0 +1,82 @@
+<template lang="pug">
+  <ComponentExample title="Results Label" id="results-label-portal" :tabs="exampleTabs" padding="-p--0">
+    p.-text(slot="example-description")
+      | Add a label to indicate the total number of results.
+    .chi-grid.-no-gutter(slot="example")
+      .chi-col.-w--12
+        .-p--3
+          chi-pagination(pages="12" current-page="3" size="xs" results="240")
```
Comment (jllr) [inline src/documentation/pages/components/pagination/examples/_portal/_results-label.vue:8]: Example is not compact

File: src/documentation/pages/components/pagination/examples/_results-label_lumen_centurylink.vue [hunk 20]
```diff
@@ -0,0 +1,125 @@
...
+        {
+          id: 'htmlblueprint',
+          label: 'HTML blueprint'
+        }
+      ],
+      codeSnippets: {
+        webcomponent: `<!-- For light backgrounds -->
+<chi-pagination pages="12" current-page="3" results="240"></chi-pagination>
+
+<!-- For light backgrounds -->
```
Comment (jllr) [inline src/documentation/pages/components/pagination/examples/_results-label_lumen_centurylink.vue:49]: Dark bg, not light. Please review code snippets

**Review by jllr — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://875de1f335446e307121ba6b3a743899aa919080-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-274-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-274/14/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://31bc8e5f1c1d0bdbf8e47076535c61b6b8a41934-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-274-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-274/15/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://875de1f335446e307121ba6b3a743899aa919080-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-274-ux-chi.kubeodc-test.corp.intranet/ (internal)

File: src/documentation/pages/components/pagination/_accessibility.vue [hunk 21]
```diff
@@ -0,0 +1,15 @@
+<template lang="pug">
+  div
```
Comment (evansito) [inline src/documentation/pages/components/pagination/_accessibility.vue:2]: AccessibilityComingSoon component

File: src/documentation/pages/components/pagination/examples/_additional-sizes_lumen_centurylink.vue [hunk 22]
```diff
@@ -0,0 +1,352 @@
...
+    </pre>
+  </ComponentExample>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+@Component({
+  data: () => {
+    return {
+      string: 'String',
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/_additional-sizes_lumen_centurylink.vue:33]: this should be removed

File: src/documentation/pages/components/pagination/examples/_base_lumen_centurylink.vue [hunk 23]
```diff
@@ -0,0 +1,116 @@
...
+    </Wrapper>
+  </ComponentExample>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+@Component({
+  data: () => {
+    return {
+      string: 'String',
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/_base_lumen_centurylink.vue:32]: same here and in further examples

File: src/documentation/pages/components/pagination/examples/_compact/_base_lumen_centurylink.vue [hunk 24]
```diff
@@ -0,0 +1,123 @@
...
+<ChiPagination :pages="3" :currentPage="2" :compact="true" />
+
+<!-- For dark backgrounds -->
+<ChiPagination :pages="3" :currentPage="2" :compact="true" :inverse="true" />`
+      }
+    };
+  }
+})
+export default class CompactBaseLumenCenturylink extends Vue {}
+</script>
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/_compact/_base_lumen_centurylink.vue:123]: no newline at end of file

File: src/documentation/pages/components/pagination/examples/_compact/_first-last-page-buttons_lumen_centurylink.vue [hunk 25]
```diff
@@ -0,0 +1,133 @@
...
+<ChiPagination :pages="3" :currentPage="2" :compact="true" :pageJumper="true" :firstLast="true" />
+
+<!-- For dark backgrounds -->
+<ChiPagination :pages="3" :currentPage="2" :compact="true" :pageJumper="true" :firstLast="true" :inverse="true" />`
+      }
+    };
+  }
+})
+export default class CompactFirstLastPageButtonsLumenCenturylink extends Vue {}
+</script>
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/_compact/_first-last-page-buttons_lumen_centurylink.vue:133]: no new line at end of file

File: src/documentation/pages/components/pagination/examples/_compact/_page-jumper_lumen_centurylink.vue [hunk 26]
```diff
@@ -0,0 +1,120 @@
...
+<ChiPagination :pages="3" :currentPage="2" :compact="true" :pageJumper="true" />
+
+<!-- For dark backgrounds -->
+<ChiPagination :pages="3" :currentPage="2" :compact="true" :pageJumper="true" :inverse="true" />`
+      }
+    };
+  }
+})
+export default class CompactPageJumperLumenCenturylink extends Vue {}
+</script>
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/_compact/_page-jumper_lumen_centurylink.vue:120]: no new line at end of file

File: src/documentation/pages/components/pagination/examples/_page-jumper_lumen_centurylink.vue [hunk 27]
```diff
@@ -0,0 +1,155 @@
...
+<ChiPagination :pages="12" :results="240" :currentPage="3" :pageSize="true" :pageJumper="true" />
+
+<!-- For dark backgrounds -->
+<ChiPagination :pages="12" :results="240" :currentPage="3" :pageSize="true" :pageJumper="true" />`
+      }
+    };
+  }
+})
+export default class PageJumperLumenCenturylink extends Vue {}
+</script>
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/_page-jumper_lumen_centurylink.vue:155]: no new line at end of file

File: src/documentation/pages/components/pagination/examples/_page-size_lumen_centurylink.vue [hunk 28]
```diff
@@ -0,0 +1,145 @@
...
+<ChiPagination :pages="12" :results="240" :currentPage="3" :pageSize="true" />
+
+<!-- For dark backgrounds -->
+<ChiPagination :pages="12" :results="240" :currentPage="3" :pageSize="true" :inverse="true" />`
+      }
+    };
+  }
+})
+export default class PageSizeLumenCenturylink extends Vue {}
+</script>
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/_page-size_lumen_centurylink.vue:145]: no new line at end of file

File: src/documentation/pages/components/pagination/examples/_additional-sizes_lumen_centurylink.vue [hunk 29]
```diff
@@ -0,0 +1,352 @@
...
+<ChiPagination :pages="5" :currentPage="3" size="xs" :inverse="true" />
+<ChiPagination :pages="5" :currentPage="3" size="sm" :inverse="true" />
+<ChiPagination :pages="5" :currentPage="3" size="md" :inverse="true" />
+<ChiPagination :pages="5" :currentPage="3" size="lg" :inverse="true" />
+<ChiPagination :pages="5" :currentPage="3" size="xl" :inverse="true" />`
+      }
+    };
+  }
+})
+export default class AdditionalSizesLumenCenturylink extends Vue {}
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/_additional-sizes_lumen_centurylink.vue:351]: I believe L in link is uppercase, i.e. CenturyLink. Please change this in all examples

File: src/documentation/pages/components/pagination/examples/_portal/_base.vue [hunk 30]
```diff
@@ -0,0 +1,79 @@
...
+    <div class="chi-pagination__end"></div>
+  </div>
+</nav>`,
+        vue: `<ChiPagination :pages="3" :currentPage="2" :compact="true" :firstLast="true" size="xs" />`
+      }
+    };
+  }
+})
+export default class CompactBasePortal extends Vue {}
+</script>
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/_portal/_base.vue:79]: no new line at end of file

File: src/documentation/pages/components/pagination/examples/_portal/_results-label.vue [hunk 31]
```diff
@@ -0,0 +1,82 @@
...
+    <div class="chi-pagination__end"></div>
+  </div>
+</nav>`,
+        vue: `<ChiPagination :pages="3" :results="240" :currentPage="2" :compact="true" :firstLast="true" size="xs" />`
+      }
+    };
+  }
+})
+export default class ResultsLabelPortal extends Vue {}
+</script>
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/_portal/_results-label.vue:82]: no new line at end of file

**Review by evansito — CHANGES_REQUESTED** [review]

File: src/documentation/pages/components/pagination/examples/index.vue [hunk 32]
```diff
@@ -0,0 +1,47 @@
...
+      <DisabledLumenCenturyLink />
+      <TruncationLumenCenturyLink />
+      <DoubleTruncationLumenCenturyLink />
+      <ResultsLabelLumenCenturyLink />
+      <PageSizeLumenCenturyLink />
+      <PageJumperLumenCenturyLink />
+      <Compact />
+      <AdditionalSizesLumenCenturyLink />
+      
+    div(v-if="$store.state.themes.theme === 'portal'")
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/index.vue:15]: v-else

File: src/documentation/pages/components/pagination/examples/_disabled_lumen_centurylink.vue [hunk 33]
```diff
@@ -0,0 +1,135 @@
+<template lang="pug">
+  <ComponentExample title="Disabled" id="disabled-lumen-centurylink" :tabs="exampleTabs" padding="-p--0">
+    .chi-grid.-no-gutter(slot="example")
+      .chi-col.-w--12.-w-md--6
```
Comment (evansito) [inline src/documentation/pages/components/pagination/examples/_disabled_lumen_centurylink.vue:4]: v-for loop could be used here

**Review by soujanreddy — APPROVED** [review]

**Review by jllr — APPROVED** [review]

**Review by janibolkvadze — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-274/20/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://cbba626aa706ed2afd497e383710639876804d6f-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-274-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Pull request merged**